Interfaces/scripts/dirac-wms-job-submit.py

```diff
@@ -30,10 +30,13 @@
 from DIRAC.Interfaces.API.Dirac                              import Dirac
 unprocessed_switches = Script.getUnprocessedSwitches()
 use_repo = False
+repo_name = ""
 for sw, value in unprocessed_switches:
   if sw.lower() in ["r", "usejobrepo"]:
     use_repo = True
-dirac = Dirac(use_repo)
+    repo_name = value
+    repo_name = repo_name.replace(".cfg", ".repo")
```

Comment:
It doesn't really matter. The only relevant thing was to change from .cfg to anything else. This particular line above is just to make sure no one uses .cfg as extension.